readme.rst

```diff
@@ -55,6 +55,17 @@ The command:
 the more standard ``make html``. For instance, ``PROJECT=azure make html`` will create the 
 azure related html pages in _build/azure.)
 
+Check compliance with contribution guides for each project 
+~~~~~~~~~~~
+Each project's documentation has its own contribution guide: 
+
+* `All-clouds <http://127.0.0.1:8000/all-clouds-how-to/contribute-to-these-docs/>`_
```

Comment:
This link with the local host IP address won't work correctly in all cases, so correct it to: https://canonical-public-cloud.readthedocs-hosted.com/en/latest/all-clouds-how-to/contribute-to-these-docs/

Reply:
Good point, thank you!